config.tf

```diff
@@ -139,16 +139,6 @@ This applies only to cloud platforms.
 EOF
 }
 
-variable "tectonic_etcd_count" {
```

Comment:
Oops. Can you pull that out into another commit, just so it's clear that this isn't part of the addition of OpenStack?

Reply:
Ah, I was a bit overzealous with the etcd removal, sorry! I've removed it.